tests/orquestra/integrations/qiskit/noise/basic_test.py

```diff
@@ -40,7 +40,7 @@ def test_get_qiskit_noise_model(self):
 
             # Then
             self.assertIsInstance(noise_model, AerNoise.NoiseModel)
-            self.assertIsInstance(coupling_map, CircuitConnectivity)
+            self.assertIsInstance(coupling_map, list)
```

Comment:
Nope, because the `coupling_map` property of the device is actually a list (despite Qiskit having CouplingMap type :man_shrugging: ). However, typing of the `get_qiskit_noise_model` function was incorrect and is now fixed in [c340c1c](https://github.com/zapatacomputing/orquestra-qiskit/pull/25/commits/c340c1cee54badc128e1791fe44ee068bb587493)

Reply:
Ok!